### PR TITLE
Usamakz patch 1

### PR DESCRIPTION
**Pull request by Usamakz**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>About</title>
+  <title>About Page</title>
```
Comment (Furqankhanzada) [inline about.html:8]:
```suggestion
  <title>About Us | Welcome to about us page where you can about Boldo</title>
```

File: about.html [hunk 2]
```diff
@@ -229,10 +229,9 @@ <h2>We aim for progress.</h2>
     <div class="container">
       <div class="newsletter-wrapper">
         <div class="news-letter-content">
-          <h2>An enterprise template to ramp up your company website</h2>
+          <h2>An enterprise template to ramp up your company website </h2>
```
Comment (Furqankhanzada) [inline about.html:232]: extra space in the end is not now required

File: about.html [hunk 3]
```diff
@@ -242,8 +241,7 @@ <h2>An enterprise template to ramp up your company website</h2>
     </div>
   </section>
 
-  <!-- new section -->
-
+  <!--  a new section -->
```
Comment (Furqankhanzada) [inline about.html:244]:
```suggestion
  <!--  footer section -->
```

**Comment by Furqankhanzada** [comment]
The PR title should meaning full.

**Comment by Furqankhanzada** [comment]
@Usamakz why you merged it? did i approved?

**Comment by Usamakz** [comment]
> The PR title should meaning full.



> @Usamakz why you merged it? did i approved?

@Furqankhanzada Bhai i don't know why this merged but i just fix all the error  and push from code studio

**Comment by Usamakz** [comment]
@Furqankhanzada bhai can i delete this branch then i create a new one
because i don't even  know this branch is merged by  my lil mistake.

**Pull request merged**